cmd/demo/node_setup.go

```diff
@@ -108,6 +108,9 @@ func (n *node) setup() error {
 func (n *node) setupContracts() (err error) {
 	var adjAddr, assAddr common.Address
 
+	// TODO: must ensure deploying node is started first, otherwise contract validation fails
+	// TODO: for better understanding of what happens, log to console when contracts are deployed
```

Comment:
We try to avoid committing TODO notes. Better create an issue if it's worth it, or fix it before giving into review.

Reply:
removed the todo notes from the source code, will consider creating issues for them

Reply:
So for the first TODO, I think that's nothing the impl can handle. The user must know how to use the node properly, so I propose to add this to the README and description of the configuration file.

For the second part, the contract addresses are already logged in the last line `	n.log.WithField("Adj", adjAddr).WithField("Asset", assAddr).Debug("Set contracts")`

Reply:
regarding first todo: the walkthrough already suggests to start alice first and then bob, so this should be ok.

regarding second todo: i still think it would be instructive to have some more information for the walkthrough user on the console. just so he gets an idea of what is going on in our framework. at the moment there is barely any information logged to the console except for what the types himself. anyhow, this can be left for another PR, as it is not closely related to this PR.